src/bindings/python/src/pyopenvino/core/remote_tensor.cpp

```diff
@@ -44,11 +48,38 @@ void regclass_RemoteTensor(py::module m) {
 
     cls.def(
         "copy_to",
-        [](RemoteTensorWrapper& self, py::object& dst) {
-            Common::utils::raise_not_implemented();
+        [](RemoteTensorWrapper& self, RemoteTensorWrapper& dst) {
+            self.tensor.copy_to(dst.tensor);
+        },
+        R"(
+        Copy tensor's data to a destination tensor. The destination tensor should have the same element type and shape.
```

Comment:
do we need a limitation on the same shape?
It's valid for ROI Tensor, but for generic tensor we can perform dst_tensor.set_shape first.

Reply:
Updated description to apply this restriction only for RoiTensors